Lib/ntpath.py

```diff
@@ -373,21 +371,18 @@ def expanduser(path):
         tilde = b'~'
     else:
         tilde = '~'
-    if not path.startswith(tilde):
+    if path[:1] != tilde:
         return path
     i, n = 1, len(path)
     while i < n and path[i] not in _get_bothseps(path):
         i += 1
 
     if 'USERPROFILE' in os.environ:
         userhome = os.environ['USERPROFILE']
-    elif not 'HOMEPATH' in os.environ:
+    elif 'HOMEPATH' not in os.environ:
```

Comment:
Is this change OK? https://www.flake8rules.com/rules/E713.html

Reply:
If it's just a stylistic change, then not really. There's a small chance that this does actually lead to a small speedup, though, since `not in` is a distinct keyword to `not` and `in`.

As we've said previously, the question you need to ask with each change is: "Does this have an impact that end users might notice?" That could mean making the code faster, or fixing bugs. _Anything else_ counts as a cosmetic change that we're unlikely to accept.

Reply:
It's faster:

```none
python -m timeit "not 'a' in 'abc'" && python -m timeit -s "'a' not in 'abc'"
20000000 loops, best of 5: 18.9 nsec per loop # before
50000000 loops, best of 5: 6.43 nsec per loop # after
# -> 2.94x faster
```

Reply:
Then it can stay :-)

Reply:
> There is an error in the microbenchmark.

Oof, I should have spotted that. Thanks.

Still, as you say, it's a small, localised change and it's a minor stylistic improvement, so no harm done.

Reply:
> There is an error in the microbenchmark.

Yeah, the code was run in the setup only.

```none
python -m timeit "not 'a' in 'abc'" && python -m timeit "'a' not in 'abc'"
20000000 loops, best of 5: 19.2 nsec per loop # before
20000000 loops, best of 5: 19.4 nsec per loop # after
# -> no difference
```